src/include/migraphx/apply_alpha_beta.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+#define MIGRAPHX_GUARD_MIGRAPHX_APPLY_ALPHA_BETA_HPP
+
+#include "migraphx/errors.hpp"
+#include <migraphx/instruction.hpp>
+#include <migraphx/instruction_ref.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/common.hpp>
+#include <migraphx/module.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+template <typename T = float>
+instruction_ref insert_apply_alpha_beta(module& m,
+                                        instruction_ref pos,
+                                        const std::vector<instruction_ref>& args,
+                                        const std::string& op_name,
+                                        T alpha = 1.0f,
+                                        T beta  = 0.0f)
```

Comment:
This function should accept that values using the `literal` class so this implementation can be moved to the cpp file. We can then provide a templated method to implicitly convert them to `literal`:

```cpp
instruction_ref insert_apply_alpha_beta_impl(module& m,
                                        instruction_ref pos,
                                        const std::vector<instruction_ref>& args,
                                        const std::string& op_name,
                                        literal alpha,
                                        literal beta);

template <class T = float>
instruction_ref insert_apply_alpha_beta(module& m,
                                        instruction_ref pos,
                                        const std::vector<instruction_ref>& args,
                                        const std::string& op_name,
                                        T alpha = 1.0f,
                                        T beta  = 0.0f)
{
    return insert_apply_alpha_beta_impl(m, pos, args, op_name, literal{alpha}, literal{beta});
}
```

Reply:
Done.  Added a source file.

Reply:
should we use:

```
T alpha = T{1},
```
here?